public-ui/src/app/data-browser/views/ehr-view/ehr-view-react.component.tsx

```diff
@@ -0,0 +1,650 @@
+                .then(results => {
+                    results.domainInfos.forEach(domain => {
+                        const thisDomain = Domain[domain.domain];
+                        if (thisDomain && thisDomain.toLowerCase() === this.props.domainId) {
+                            const ehrDomain = domain;
+                            const subTitle = 'Keyword: ' + this.props.searchTerm;
+                            const title = ehrDomain.name;
+                            const totalParticipants = ehrDomain.participantCount;
+                            const numPages = Math.ceil(ehrDomain.standardConceptCount / 50);
+                            this.setState({
+                                domain: domain,
+                                title: title,
+                                subTitle: subTitle,
+                                totalParticipants: totalParticipants,
+                                numPages: numPages,
+                                totalResults: ehrDomain.standardConceptCount,
+                            }, () => {
+                                this.getTopConcepts();
+                            });
+                        }
+                });
+                }).catch(e => {
+                        console.log(e, 'error');
+           });
+   }
+
+   processSearchResults(results) {
+        const {searchWord, top10Results, currentPage, domain} = this.state;
+        results.items.filter(
+            x => PM_CONCEPTS.indexOf(x.conceptId) === -1);
+        results.items.sort((a, b) => b.countValue - a.countValue);
+        let medlineTerm = searchWord ? searchWord : '';
+        if (results.matchType === MatchType.ID || results.matchType === MatchType.CODE) {
+              medlineTerm = results.matchedConceptName;
+        }
+        const medlinePlusLink = 'https://vsearch.nlm.nih.gov/vivisimo/cgi-bin/query-meta?v%3Aproject=' +
+            'medlineplus&v%3Asources=medlineplus-bundle&query='
+            + medlineTerm;
+        for (const concept of results.items) {
+            concept['synonymString'] = concept.conceptSynonyms.join(', ');
+            concept['drugBrands'] = concept.drugBrands;
+            if (domain.domain.toLowerCase() === 'measurement') {
+                concept.graphToShow = GraphType.Values;
+            } else {
+                concept.graphToShow = GraphType.BiologicalSex;
+            }
+        }
+        let standardConcepts = [];
+        if (results.standardConcepts) {
+            standardConcepts = results.standardConcepts;
+        }
+        let topResults = top10Results;
+        if (currentPage === 1) {
+            topResults = results.items.slice(0, 10);
+        }
+        this.setState({concepts: results.items, standardConcepts: standardConcepts, top10Results: topResults,
+        loading: false, medlineTerm: medlineTerm, medlinePlusLink: medlinePlusLink});
```

Comment:
Nit: can simplify this by dropping the if statements and setting `standardConcepts` and  `top10Results` inline:
```suggestion
        this.setState({
          concepts: results.items,
          standardConcepts: results.standardConcepts || [],
          top10Results: currentPage === 1 ? results.items.slice(0, 10) : top10Results,
          loading: false,
          medlineTerm: medlineTerm,
          medlinePlusLink: medlinePlusLink
        });
```

Reply:
changed this thanks.